x-pack/plugins/ml/public/_app.scss

```diff
@@ -0,0 +1,30 @@
+// ML has app specific coloring for it's various warning levels. 
+// These are used almost everywhere.
+
+.ml-icon-severity-critical,
+.ml-icon-severity-major,
+.ml-icon-severity-minor,
+.ml-icon-severity-warning,
+.ml-icon-severity-unknown {
+  text-shadow: 1px 1px 1px $euiColorLightestShade;
```

Comment:
Just a note for review. I avoided touching shadows on this pass. A lot of ML uses text shadows and drops that I didn't have a quick 1 to 1 parallel for. Mostly I'd change the color to match against the EUI counterpart and then move forward. Rest can be handled on a 2nd pass.